public/app/features/dashboard/components/DashboardSettings/ListNewButton.tsx

```diff
@@ -2,20 +2,14 @@ import React, { ButtonHTMLAttributes } from 'react';
 import { Button, useStyles } from '@grafana/ui';
 import { GrafanaTheme } from '@grafana/data';
 import { css } from '@emotion/css';
-import { selectors } from '@grafana/e2e-selectors';
 
 export interface Props extends ButtonHTMLAttributes<HTMLButtonElement> {}
 
 export const ListNewButton: React.FC<Props> = ({ children, ...restProps }) => {
   const styles = useStyles(getStyles);
   return (
     <div className={styles.buttonWrapper}>
-      <Button
-        aria-label={selectors.pages.Dashboard.Settings.Annotations.List.newButton}
-        icon="plus"
-        variant="secondary"
-        {...restProps}
-      >
+      <Button icon="plus" variant="secondary" {...restProps}>
```

Comment:
Doesn't aria-label work here anymore?

Reply:
It looks like if we use an `aria-label` here, it would break the FE tests at https://github.com/grafana/grafana/blob/main/public/app/features/dashboard/components/DashboardSettings/AnnotationsSettings.test.tsx#L243 where we are trying to get the button by the text `new query` because RTL would expect the `aria-label` text instead e.g. `getByRole('button', { name: 'Annotations list new annotation button' }` instead of `getByRole('button', { name: '/new query/i' }`

I'n not sure updating the FE tests to use the `aria-label` instead would be better if we have actual text available ('New query') for the button that may be more user friendly, if that makes sense? Or if you think it's fine to replace with the selector here instead then happy to revert

Reply:
Good points and I'm not sure, really. I prefer both RTL and e2e use the same selectors but I'm don't have superstrong opinions about it.